.github/workflows/ci.yml

```diff
@@ -9,22 +9,19 @@ on:
 jobs:
   pod-lint:
     name: Pod Lint
-    runs-on: macOS-12
+    runs-on: macos-13
     steps:
-      - uses: ruby/setup-ruby@v1
-        with:
-          ruby-version: '2.7.6'
```

Comment:
we no longer need these lines because we're using a built-in ruby version rather than a custom one.